crates/proto/src/rr/rdata/caa.rs

```diff
@@ -134,15 +134,36 @@ impl CAA {
         self.tag = tag;
     }
 
-    /// a potentially associated value with the property tag, see struct documentation
-    pub fn value(&self) -> &Value {
+    /// A value associated with the property tag, see struct documentation
+    pub fn value(&self) -> &[u8] {
```

Comment:
This is a semver-incompatible change. I hoped we would be able to keep main semver-compatible for a while... do you think there's a sensible version of these changes that is semver-compatible?

Reply:
Yeah, we'll just have to keep a second copy of the value around in the enum

Reply:
Want to try that so we can have a look?